static/js/components/UserInfoCard.js

```diff
@@ -26,8 +27,47 @@ export default class UserInfoCard extends React.Component {
     </div>
   );
 
+  renderAboutMeSection: Function = (
+    profile: Profile, toggleShowAboutMeDialog: Function
+  ): React$Element<*> => {
+    let aboutMeContent = userPrivilegeCheck(
+      profile,
+      () => [
+        <h3 key="heading">About Me</h3>,
+        <div className="bio placeholder" key="bio-placeholder">
+          Write something about yourself, so others can learn a bit about you.
+        </div>
+      ],
+      null
+    );
+    if (profile.about_me) {
+      aboutMeContent = [
+        <h3 key="heading">About Me</h3>,
+        <div className="bio" key="bio">{profile.about_me}</div>
+      ];
+    }
+
+    return (
+      <div className="profile-form-row">
+        <div className="about-me">{aboutMeContent}</div>
+        <div className="edit-about-me-holder">
+          {
+            userPrivilegeCheck(
+              profile,
+              () => <IconButton name="edit about me section" onClick={toggleShowAboutMeDialog}/>
```

Comment:
I guess this works -- it renders fine, and the screenreader picks up the full text correctly. It's not the solution I expected, but if it works, then great!